locust/from_apache.py

```diff
@@ -1,54 +1,60 @@
-import argparse
-import os
 import re
-import random
 from locust import HttpUser, task, between
 
 
 def generate_url(apache_urls_path='results.txt'):
     f = open(apache_urls_path, 'r')
     for line in f:
         yield line
```

Comment:
As an alternative, on startup (or it can be pre-generated), fetch names of datasets, groups, orgs from the API. Then, instead of just reading the line directly as-is, we match the URL pattern, similar to what we're doing in `get_name` and replace the id/name with a real one that exists.